### PR TITLE
Implementation of Unit Test for widgets

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
* Key values were added on each widget to perform the tests.
* Only for widgets:
  - movie_event_initial
  - movie_event_error
  - movie_event_empty
  They were added in a single test.

In the following image I will show the results of the tests.
test_results

### CODE REVIEW

File: lib/src/widgets/movie_backdrop_image.dart [hunk 1]
```diff
@@ -10,36 +10,150 @@ class MovieBackdropImage extends StatelessWidget {
...
+                              Dimension.genreCardOpacity,
+                            ),
+                            fontSize: Dimension.genreCardFontSize,
+                          ),
+                          textAlign: TextAlign.center,
+                        ),
+                      ],
+                    ),
+
+ */
```
Comment (MarcosCGlobant) [inline lib/src/widgets/movie_backdrop_image.dart]: if you aren't using this code, remove it please

File: lib/src/widgets/movie_card.dart [hunk 2]
```diff
@@ -55,3 +188,4 @@ class MovieCard extends StatelessWidget {
     );
   }
 }
+*/
```
Comment (MarcosCGlobant) [inline lib/src/widgets/movie_card.dart:191]: if your aren't using this commented code, please remove it

Comment (MarcosCGlobant) [inline lib/src/widgets/movie_card.dart]: if you aren't using this commented code, please remove it

File: lib/src/widgets/movie_backdrop_image.dart (hunk 1 above)
Comment (FranciscoGaraventa) [reply lib/src/widgets/movie_backdrop_image.dart]: Done

File: lib/src/widgets/movie_card.dart (hunk 2 above)
Comment (FranciscoGaraventa) [reply lib/src/widgets/movie_card.dart]: Done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**